tests/sidetrail/count_success.pl

```diff
@@ -22,20 +22,22 @@
 
 sub  trim { my $s = shift; $s =~ s/^\s+|\s+$//g; return $s };
 
-if (@ARGV != 2) {
-    die "usage: count_success.pl expected_success expected_failures";
+if (@ARGV != 3) {
+    die "usage: count_success.pl expected_success expected_failures filename";
 }
 
 my $expected_success = shift;
 my $expected_failure = shift;
+my $filename = shift;
 my @undefined_functions = ();
 my %allowed_undefined = ("__CONTRACT_invariant" => 1,
 			 "malloc" => 1,
 			 "nondet" => 1);
 
 my $verified = 0;
 my $errors = 0;
-while (my $line = <STDIN>){
+open (FILE, $filename) or die "Can't open $filename $!";
+while (my $line = <FILE>){
     print $line;
```

Comment:
One small thing: since this prints the line, we end up with each test appearing twice in the logs, which makes the logs larger than they need to be.  We could remove this and make the logs 1/2 the size.

Reply:
Created #1536 is track this. I'll merge this PR so others can rebase, and submit the new PR.